annotation-processor/src/test/java/org/hibernate/validator/ap/testmodel/annotationparameters/InvalidDigitsParameters.java

```diff
@@ -23,7 +23,27 @@
 	@Digits( integer = -5, fraction = -3 )
 	private BigDecimal decimal3;
 
-	@Digits.List({ @Digits(integer = -5, fraction = -3), @Digits(integer = 5, fraction = -3) })
+	@Digits.List( {@Digits( integer = -5, fraction = -3 ), @Digits( integer = 5, fraction = -3 )} )
 	private BigDecimal decimal4;
 
+	public InvalidDigitsParameters(
+			@Digits( integer = -1, fraction = 3 ) BigDecimal decimal1,
```

Comment:
Thanks for the link on formatters - it worked just great! Also importing of formatter in Intellij can be done through the UI rather than copying files :)

One day I'll create a PR and you wouldn't need to clean up anything after me :)